.github/workflows/latest-release.yml

```diff
@@ -1,31 +1,70 @@
+name: Test KubeArmor image before pushing
 on:
   push:
     # Sequence of patterns matched against refs/tags
     branches:
       - 'main'
 
-name: Latest version release
+env: 
+  TEST_TAG: kubearmor/kubearmor:test
 
 jobs:
   build:
     name: Create KubeArmor Release - 18.04
-    if: github.repository == 'kubearmor/KubeArmor'
+    # if: github.repository == 'kubearmor/KubeArmor'
     runs-on: ubuntu-18.04
     steps:
       - name: Checkout KubeArmor code
         uses: actions/checkout@v2
 
       - name: Generate KubeArmor artifacts
-        run: GITHUB_SHA=$GITHUB_SHA ./KubeArmor/build/build_kubearmor.sh
+        run: GITHUB_SHA=$GITHUB_SHA ./KubeArmor/build/build_kubearmor.sh test
+
+      - name: Setup Enviroment
+        run: |        
+          # create a single-node K3s cluster
+          curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" INSTALL_K3S_EXEC="--flannel-backend=none --cluster-cidr=192.168.0.0/16 --disable-network-policy --disable=traefik" sh -
+
+          #setting a variable environment
+          mkdir ~/.kube/
+          cp /etc/rancher/k3s/k3s.yaml ~/.kube/config 
+
+          # install Calico-Operator
+          kubectl create -f https://docs.projectcalico.org/manifests/tigera-operator.yaml
+
+          # install Calico-Manifest
+          kubectl apply -f https://docs.projectcalico.org/manifests/calico.yaml
+            
+      - name: Run KubeArmor 
+        run: |
+          kubectl apply -f KubeArmor/build/kubearmor-test-k3s.yaml
+          kubectl get pods --all-namespaces 
+          # kubectl rollout status
+          while [[ $(kubectl get pods -l kubearmor-app=kubearmor -n kube-system -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do echo "waiting for pod" && sleep 1; done
+
+          #kubectl wait --for=condition=Ready pods --all --all-namespaces
```

Comment:
do we need to keep these lines even though they are commented out?

Reply:
No, I can remove them. 
Actually these command were used to find if the kubearmor is ready and then only start the next job so that the tests doesn't fail due to the unavailability of the logs file. But even after verifing the kubearmor's working, few tests were still failing so I added the sleep command to fix the issue. But it is not a permanent fix and we need to find another way to verify that everything runs smooth before performing the tests.